src/components/SettingsDialog.vue

```diff
@@ -81,6 +81,86 @@
                 </ul>
             </fieldset>
         </NcAppSettingsSection>
+        <NcAppSettingsSection
+            id="settings-info-blocks"
+            :title="t('cookbook', 'Info blocks')"
+            class="app-settings-section"
+        >
+            <fieldset>
+                <legend class="settings-info-blocks__legend">
+                    {{
+                        t(
+                            "cookbook",
+                            `Control which blocks of information are shown in
+                            the recipe view. If you do not use some features
+                            and find them distracting, you may hide them.`
+                        )
+                    }}
```

Comment:
This will not be translatable.

Reply:
Because of the template literal? I was trying to avoid going over the line length.

Reply:
The template literal is no problem as far as i know.

The thing is that the translation ci script users a regex that does not allow newlines. So, the complete string must be in one line. Otherwise, the string cannot be translated.

One has to add an exception for prettier to avoid the line to be changed. 

I asked for a fox but they fear to break other apps.

Reply:
Thank you for clarifying. I hope it is fixed.